src/components/Greeting.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react';
```

Comment:
- [ ] Currently, an error displays on the frontend saying "message not found". Kindly fix this, please. Your app should display a random greeting every time a user refreshes the page.
![image](https://user-images.githubusercontent.com/94551484/228977081-f76c9bcc-6282-408e-afed-5d0a4634b9d5.png)